crates/forge/src/running.rs

```diff
@@ -103,6 +138,10 @@ pub(crate) fn blocking_run_from_test(
             &send_shut_down,
         );
 
+        if send.is_closed() {
+            return Ok(TestCaseSummary::Skipped {});
+        }
```

Comment:
To fix out tests, other wise no way to test --exit-fist flag

Reply:
This makes no sense, it basically means our tests describe a property poorly, because they don't test if execution time is reduced (which should be the goal of exit_first) but what is printed.

Reply:
👍 so I think that I will remove this code and ignore those tests. + Create an issue on proper testing of --exit-first flag.

Reply:
Testing based on execution time sounds like a bad idea. Most likely it will still fail periodically in case CI decides to slow down for whatever reason etc.

Reply:
@cptartur Disagreed, If you make one test run basically infinitely, and one run in a seconds it will always work. What matters is the stastistical significance of the time difference.

Reply:
I created the [issue](https://github.com/orgs/foundry-rs/projects/3/views/2?pane=issue&itemId=43725188)